### PR TITLE
Refactor theming structure

### PR DESCRIPTION
**Pull request by jamesricky**
This keeps the `getThemeOptions` file more compact for a better overview.

- Name functions with "get" prefix
- Group props and overrides per package
- Name overrides/props groups according to admin-package
- Remove "CometAdmin" prefix from overrides/props files and functions

### CODE REVIEW

File: packages/admin-theme/src/CometAdminColorPickerOverrides/colorPicker.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { CometAdminColorPickerClassKeys } from "@comet/admin-color-picker";
 import { StyleRules } from "@material-ui/styles/withStyles";
 
-export const cometAdminColorPickerOverrides = (): StyleRules<{}, CometAdminColorPickerClassKeys> => ({
+export const getCometAdminColorPickerOverrides = (): StyleRules<{}, CometAdminColorPickerClassKeys> => ({
```
Comment (manuelblum) [inline packages/admin-theme/src/CometAdminColorPickerOverrides/colorPicker.ts:4]: do we need the CometAdmin prefix on all Overrides or wouldn't it be better to reduce it to e.g.: `getColorPickerOverrides()` or `colorPickerOverrides()` for better readability.

Comment (jamesricky) [reply packages/admin-theme/src/CometAdminColorPickerOverrides/colorPicker.ts:4]: I think you're right, including the full component-name in the function-name probably doesn't bring any benefit.

**Comment by nsams** [comment]
is `get` a good prefix?

doesn't somehow imply `get` that there is also `set`?

`create` would not do that, however I would expect something factory like with options.

Why is it a fn anyway? Couldn't we just export the object that is used as-is and use `create` prefix for factories (where we have options)?

**Comment by jamesricky** [comment]
> is `get` a good prefix?
> 
> doesn't somehow imply `get` that there is also `set`?
> 
> `create` would not do that, however I would expect something factory like with options.

To me, `get` only implies, that a value is returned... and `create` sounds more like something is created, but not returned.
Does it make more sense, to remove the `get` again? I just added it, because `cometAdminOverrides` seems like an incomplete function-name to me 😄

> Why is it a fn anyway? Couldn't we just export the object that is used as-is and use `create` prefix for factories (where we have options)?

There will be a `darkMode: boolean` parameter added in the future, which needs to be passed into the function.

**Comment by nsams** [comment]
> To me, `get` only implies, that a value is returned...

yes, you can argue that.

> and `create` sounds more like something is created, but not returned.

No, create is used for factory style function.

I guess we will wait for other opinions :)

**Comment by rabengraph** [comment]
`get...Props` and `get...Overrides` is good!

> create would not do that, however I would expect something factory like with options.

exactly, a factory would expect arguments for the creation-process, which is not the case here.

> doesn't somehow imply get that there is also set?

Getter only for readonly values is perfectly fine to me.

**Review by kaufmo — APPROVED** [review]

**Review by nsams — APPROVED** [review]

**Pull request merged**